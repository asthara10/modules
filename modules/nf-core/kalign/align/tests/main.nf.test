nextflow_process {

    name "Test Process KALIGN_ALIGN"
    script "../main.nf"
    process "KALIGN_ALIGN"

    tag "modules"
    tag "modules_nfcore"
    tag "kalign"
    tag "kalign/align"

    test("test_kalign_align") {


        when {
            process {
                """
                
                input[0] = [
        [ id:'test' ], // meta map
        file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
    ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
