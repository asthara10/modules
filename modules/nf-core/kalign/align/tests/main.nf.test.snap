{
    "versions": {
        "content": [
            [
                
            ]
        ],
        "timestamp": "2023-11-24T14:20:56.323243"
    }
}